src/widgets/paragraph.rs

```diff
@@ -196,6 +196,18 @@ impl<'a> Paragraph<'a> {
         self
     }
 
+    /// Set the text alignment for the given paragraph
+    /// 
```

Comment:
There's a extra space at the end of this line and the other empty line below that rustfmt will delete.
```suggestion
    ///
```
run the formatter locally using:
```shell
cargo +nightly fmt --all
```